package.json

```diff
@@ -127,27 +127,27 @@
   },
   "devDependencies": {
     "@types/follow-redirects": "^1.14.1",
-    "@types/glob": "^7.2.0",
-    "@types/mocha": "^9.1.0",
-    "@types/node": "^17.0.23",
+    "@types/glob": "^8.1.0",
+    "@types/mocha": "^10.0.1",
+    "@types/node": "^18.15.3",
     "@types/sinon": "^10.0.11",
     "@types/vscode": "^1.65.0",
     "@vscode/test-electron": "^2.1.3",
-    "glob": "^7.1.6",
-    "mocha": "^9.2.2",
+    "glob": "^9.3.0",
+    "mocha": "^10.2.0",
     "prettier": "^2.6.2",
-    "sinon": "^13.0.2",
+    "sinon": "^15.0.2",
     "ts-loader": "^9.3.0",
-    "typescript": "^4.6.3",
-    "vsce": "^2.11.0",
+    "typescript": "^5.0.2",
     "vscode-nls-dev": "^4.0.1",
+    "vscode-tmgrammar-test": "^0.1.1",
     "webpack": "^5.76.0",
-    "webpack-cli": "^4.9.2"
+    "webpack-cli": "^5.0.1"
   },
   "dependencies": {
+    "@vscode/vsce": "^2.18.0",
```

Comment:
so it is going from `vsce` as devDependencies to `@vsce/vsce` as dependencies?

Reply:
Correct. There's a deprecation notice on the `vsce` package page: https://www.npmjs.com/package/vsce.